schemas/multimodal/v1/inventory.proto

```
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package fiftyone.multimodal.schemas.v1;
+
+import "common.proto";
```

Buf (1.68.4)

[error] imported file does not exist

(COMPILE)